<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lb_time">
      <property name="styleSheet">
       <string notr="true">font: 36pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Время</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="tb_round_time">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Segoe UI&quot;;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_start_stop">
      <property name="text">
       <string>Старт</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_round">
      <property name="text">
       <string>Круг</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_clear">
      <property name="text">
       <string>Отчистить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_exit">
      <property name="text">
       <string>Выход</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>336</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pb_exit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>435</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
